### DISCUSSION TITLE
Problema Syntax error

### DISCUSSION THREAD

**Discussion by JassielPadua99** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#213] [Q&A]
Hola buenas noches, tengo una pregunta intente correr uno de los programas de ejemplo de Html, pero me aparece un syntax error en la parte de (<!DOCTYPE html>). Creen que me puedan ayudar o saber porque me sale asi. por favor 
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Selectores</title>
    <style>
        *{
            font-family: Arial, Helvetica, sans-serif;
        }

        body{
            background-color: aqua;
        }

        p{
            color: red;
            text-align: center;
        }

        .titulo{
            color: rgb(200,150,220);
            text-align: right;
        
        }
    

        .izquierda{
            color: green;
        }

        p.izquierda{
            text-decoration: underline;
            text-align: left;
        }

        .contraste{
            background-color: black;
        }

        h1, h2, p{
            font-weight: bold;
        }

        /* Comentario CSS */

    </style>
</head>
<body>

    <h1>Header simple</h1>
    <h2 id="titulo" class="izquierda">Header con id Titulo</h2>
    <p>Parrafo simple</p>

    <h3 class="izquierda">Prueba clase Header 3</h3>
    <p class="izquierda">Prueba clase parrafo</p>
    <h3 class="izquierda contraste">Prueba multiples clases parrafo</h3>
    
</body>
</html>

**Comment by hackit92** [comment]
Perdón, estoy entendiendo que añadiste todo el css en el html?

**Comment by JuanSalvador73** [comment]
¿Como armaste el HTML?

**Comment by JuanSalvador73** [comment]
No me marcó error, yo creo te faltó incluir el tag de apertura "<html>"  y cierre "</html>" o algo parecido. Hice la prueba con el código completo y lo abre sin problemas.

<html>
</html>

![image](https://user-images.githubusercontent.com/99113565/156504616-1999e73d-f244-4af0-a219-52cb4d8bd031.png)

Checa cada línea de tu código, seguramente te faltó algo.

**Comment by doguedogue** [comment]
Se ve bien, no le veo errores en la consola, separaste el style en el head y el contenido en él body? 👌


![image](https://user-images.githubusercontent.com/23409026/156509288-0ccc1e35-c8a7-4294-94c5-fd5ce0f041e8.png)


```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Prueba</title>
    <title>Selectores</title> 
    
    <style> 
        *{ 
            font-family: Arial, Helvetica, sans-serif; 
        }

        body{
            background-color: aqua;
        }
    
        p{
            color: red;
            text-align: center;
        }
    
        .titulo{
            color: rgb(200,150,220);
            text-align: right;
        
        }
        
        .izquierda{
            color: green;
        }
    
        p.izquierda{
            text-decoration: underline;
            text-align: left;
        }
    
        .contraste{
            background-color: black;
        }
    
        h1, h2, p{
            font-weight: bold;
        }
    
        /* Comentario CSS */
    </style>    
</head>
<body>
    
    <h1>Header simple</h1>
    <h2 id="titulo" class="izquierda">Header con id Titulo</h2>
    <p>Parrafo simple</p>
    
    <h3 class="izquierda">Prueba clase Header 3</h3>
    <p class="izquierda">Prueba clase parrafo</p>
    <h3 class="izquierda contraste">Prueba multiples clases parrafo</h3>
    
</body>
</html>
```

**Comment by JassielPadua99** [comment]
![Problema](https://user-images.githubusercontent.com/99385801/156510977-7245f886-ec02-4909-bc21-3d617b457dae.png)

**Reply by JonathanJRodriguez** [reply]
Buenos días compañero,

No sé si ya resolviste tu problema o si este sea el caso, pero los html se abren con el navegador, veo que el error que dices que te marca es en la consola de python, que no se está usando para estos temas.
Para abrir un archivo html simplemente dale doble click en donde lo tengas guardado y debería abrirte tu navegador web por default con tu página cargada.

Saludos,

**Comment by JassielPadua99** [comment]
estuve probando y viendo pero me sigue apareciendo el error en la primera linea, y nose a que se deba

**Comment by JuanSalvador73** [comment]
En donde dice título en tu código le pusiste un punto de clase , cuando debe de ir un numeral # de ID porque es id:

![image](https://user-images.githubusercontent.com/99113565/156520803-e0345398-e1c8-42c8-8753-f324fefae424.png)

Checa el título en el H2 en el HTML:

![image](https://user-images.githubusercontent.com/99113565/156521316-50e9b231-1e7e-4f3a-bb4b-2d95f29e5bd3.png)


También puede ser que usaras paréntesis en lugar de corchetes en la clase "titulo", pero a simple vista se ve que si son corchetes. Necesitaría ver tu código completo.

**Comment by linettelopb** [comment]
1. Por lo que veo estas intentando correrlo desde la terminal y eso no es necesario solo guarda tu archivo ve a tu carpeta donde tengas el archivo html y abrelo y listo en automatico te lo mostrara en el navegador predeterminado.
2. Si tienes un detalle al declarar estilo a titulo como menciona el compañero ya que segun tu codigo lo esta identificando con un id y por ende en el css deberia ir (#titulo) si fuera una clase si estaria correcto el (.titulo) aunque lo tengas incorrecto esto no genera error ya que simplemente ese elemento h2 no tiene estilos definidos por el id.

Espero esto te ayude y puedas avanzar.

**Comment by Ingrzs** [comment]
Instala la extensión open in browser, después de eso da click derecho encima del código en cualquier parte y elije la opción open in the browser (para abrirlo mediante el navegador elije con que navegador lo quieres abrir) y listo se abrirá la pagina.
Un consejo esto ya no es como python, todo codigo de html se abre o "ejecuta" en el navegador web.
Ubica donde tienes guardado tu archivo ese que estas haciendo y dale clik derecho y elije el navegador de tu elección y listo .
